Index: src/scripts/relnotes.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/scripts/relnotes.py b/src/scripts/relnotes.py
--- a/src/scripts/relnotes.py	(revision 00dcbf0c0bd42acc343dd4a3073aa87b947f3e9a)
+++ b/src/scripts/relnotes.py	(revision d365263f8700dc90b5649ddaca72ff61686f5576)
@@ -32,7 +32,7 @@
 try:
   from xml.etree import ElementTree
 except ImportError:
-  print "Building release notes is not supported on this platform."
+  print("Building release notes is not supported on this platform.")
   sys.exit(0)
 
 
@@ -41,14 +41,14 @@
 
 def print_usage(prgm_name):
   """ Print the usage for this program """
-  print "Usage: " + prgm_name + " <target-dir> <git-src> <commit-range> " \
-      + "<newversion> <oldversion>"
-  print ""
-  print "  <target-dir>: Directory where release notes should be written to."
-  print "  <git-src>: Root of the git repository to collect info from."
-  print "  <commit-range>: What set of commits form this release."
-  print "  <newversion>: The version number to print in the release notes."
-  print "  <oldversion>: The previous release version number."
+  print("Usage: " + prgm_name + " <target-dir> <git-src> <commit-range> " \
+      + "<newversion> <oldversion>")
+  print ("")
+  print("  <target-dir>: Directory where release notes should be written to.")
+  print("  <git-src>: Root of the git repository to collect info from.")
+  print("  <commit-range>: What set of commits form this release.")
+  print("  <newversion>: The version number to print in the release notes.")
+  print("  <oldversion>: The previous release version number.")
 
 
 def get_log(git_dir, commit_range):
@@ -224,7 +224,7 @@
     return 0
 
   if len(argv) < NUM_ARGS:
-    print "Missing required argument(s). Try " + argv[0] + " -h"
+    print("Missing required argument(s). Try " + argv[0] + " -h")
     return 1
 
   target_dir = os.path.abspath(os.path.expanduser(argv[1]))
Index: sqoop-patch-review.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/sqoop-patch-review.py b/sqoop-patch-review.py
--- a/sqoop-patch-review.py	(revision 00dcbf0c0bd42acc343dd4a3073aa87b947f3e9a)
+++ b/sqoop-patch-review.py	(revision d365263f8700dc90b5649ddaca72ff61686f5576)
@@ -65,7 +65,7 @@
   elif (cmd_exists("rbt") == 0):
     post_review_tool = "rbt post"
   else:
-    print "please install RBTools"
+    print("please install RBTools")
     sys.exit(1)
 
   patch_file=tempfile.gettempdir() + "/" + opt.jira + ".patch"
@@ -86,16 +86,16 @@
   p_then.close()
 
   if branch_now != branch_then:
-    print 'ERROR: Your current working branch is from an older version of ' + opt.branch + '. Please rebase first by using git pull --rebase'
+    print('ERROR: Your current working branch is from an older version of ' + opt.branch + '. Please rebase first by using git pull --rebase')
     sys.exit(1)
 
   git_configure_reviewboard="git config reviewboard.url https://reviews.apache.org"
-  print "Configuring reviewboard url to https://reviews.apache.org"
+  print("Configuring reviewboard url to https://reviews.apache.org")
   p=os.popen(git_configure_reviewboard)
   p.close()
 
   git_remote_update="git remote update"
-  print "Updating your remote branches to pull the latest changes"
+  print("Updating your remote branches to pull the latest changes")
   p=os.popen(git_remote_update)
   p.close()
 
@@ -113,34 +113,34 @@
   if opt.testing:
     rb_command=rb_command + " --testing-done=" + opt.testing
   if opt.debug:
-    print rb_command
+    print(rb_command)
   p=os.popen(rb_command)
   rb_url=""
   for line in p:
-    print line
+    print (line)
     if line.startswith('http'):
       rb_url = line
     elif line.startswith("There don't seem to be any diffs"):
-      print 'ERROR: Your reviewboard was not created/updated since there was no diff to upload. The reasons that can cause this issue are 1) Your diff is not checked into your local branch. Please check in the diff to the local branch and retry 2) You are not specifying the local branch name as part of the --branch option. Please specify the remote branch name obtained from git branch -r'
+      print('ERROR: Your reviewboard was not created/updated since there was no diff to upload. The reasons that can cause this issue are 1) Your diff is not checked into your local branch. Please check in the diff to the local branch and retry 2) You are not specifying the local branch name as part of the --branch option. Please specify the remote branch name obtained from git branch -r')
       p.close()
       sys.exit(1)
     elif line.startswith("Your review request still exists, but the diff is not attached") and not opt.debug:
-      print 'ERROR: Your reviewboard was not created/updated. Please run the script with the --debug option to troubleshoot the problem'
+      print('ERROR: Your reviewboard was not created/updated. Please run the script with the --debug option to troubleshoot the problem')
       p.close()
       sys.exit(1)
   if p.close() != None:
-    print 'ERROR: reviewboard update failed. Exiting.'
+    print('ERROR: reviewboard update failed. Exiting.')
     sys.exit(1)
   if opt.debug:
-    print 'rb url=',rb_url
+    print('rb url=',rb_url)
 
   git_command="git format-patch " + opt.branch + " --stdout > " + patch_file
   if opt.debug:
-    print git_command
+    print(git_command)
   p=os.popen(git_command)
   p.close()
 
-  print 'Creating diff against', opt.branch, 'and uploading patch to JIRA',opt.jira
+  print('Creating diff against', opt.branch, 'and uploading patch to JIRA',opt.jira)
   jira=get_jira()
   issue = jira.issue(opt.jira)
   attachment=open(patch_file)
@@ -149,9 +149,9 @@
 
   comment="Created reviewboard "
   if not opt.reviewboard:
-    print 'Created a new reviewboard',rb_url,
+    print('Created a new reviewboard',rb_url,)
   else:
-    print 'Updated reviewboard',rb_url
+    print('Updated reviewboard',rb_url)
     comment="Updated reviewboard "
 
   comment = comment + rb_url + ' against branch ' + opt.branch
